buildenv/jenkins/JenkinsfileBase

```diff
@@ -12,101 +11,82 @@ def getBuildList() {
 	return TESTPROJECT
 }
 
-def getJCK(){
-	dir("$WORKSPACE/jck/"){
-		def jck_suite = "runtime"
-		def jck_version = "0"
-		if (env.JCK_VERSION == "jck8b") {
-			jck_version = "8b"
-		} else if (env.JCK_VERSION == "jck9") {
-			jck_version = "9"
-		} else if (env.JCK_VERSION == "jck10") {
-			jck_version = "10"
+def archiveTestBinaries() {
+	dir("$WORKSPACE") {
+		sh "tar -zcf test-binaries.tar.gz ./openjdk-tests ./jvmtest ./openjdkbinary"
```

Comment:
This basically get the whole workspace tar-ed, how big it will be? If tests don't need the test source code /jvmtest should be enough. Otherwise with build.xml updated /openjdk-tests should be right directory to tar-ed.
There is also no need to tar /openjdkbinary again, which has sdk tar-ed.

Reply:
archiveTestBinaries() is called after compile stage. That is all materials related with specific test type have been staged. Say regression test, there will be the whole openjdk repo. I don't think it is pretty small. Though it may not be a concern for now if jck tests material is small and only jck tests will be enabled parallelism. It stays as a concern for later.

jdk is a tar under ./jdkbinary I really don't think it's necessary to re-tar(which will tar all the unzipped jdk and zipped jdk). Can update archiveArtifacts artifacts: '**/test-binaries.tar.gz, yoursdkfilename', fingerprint: true, allowEmptyArchive: false 

and get test.tar and sdktar by copyArtifacts filter: 'test.tar, sdktar', fingerprintArtifacts: true, projectName: 'gsdgsg'.